test/controllers/tasks_controller_test.rb

```diff
@@ -28,28 +30,29 @@
   # Unskip these tests for Wave 2
   describe "show" do
     it "can get a valid task" do
-      skip
+      # skip
       # Act
       get task_path(task.id)
 
       # Assert
       must_respond_with :success
     end
 
-    it "will redirect for an invalid task" do
-      skip
+    it "will respond with 302 for an invalid task" do
+      # skip
       # Act
-      get task_path(-1)
+      expect {
+        get task_path(-1)
+      }.wont_change "Task.count"
 
       # Assert
-      must_respond_with :redirect
-      expect(flash[:error]).must_equal "Could not find task with id: -1"
+      must_respond_with 302
```

Comment:
I suggest using the rails symbols for response codes instead of using the numbers.  It's just more readable.